Makefile

```diff
@@ -4,19 +4,8 @@ KUEUE_TMP_PATH=/tmp/xpk_tmp/kueue
 KUBECTL_VERSION := $(shell curl -L -s https://dl.k8s.io/release/stable.txt)
 OS := $(shell dpkg --print-architecture)
```

Comment:
Can we rename it, please? Because it confused me. I think OS should be darwin/linux. PLATFORM arm64/amd64.